CMakeLists.txt

```diff
@@ -1,18 +1,15 @@
 cmake_minimum_required(VERSION 3.15)
-cmake_policy(SET CMP0091 NEW)
-project(novatel-edie)
+include(cmake/third_party.cmake)
 
-enable_testing()
+project(novatel-edie VERSION 3.3.7 LANGUAGES CXX)
 
+option(BUILD_TESTS "Build tests" ON)
+option(BUILD_EXAMPLES "Build examples" ON)
+option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
+option(CMAKE_POSITION_INDEPENDENT_CODE "Set -fPIC" ON)
```

Comment:
`CMAKE_POSITION_INDEPENDENT_CODE` is a CMake variable that enables the addition of `-fPIC` on non-Windows builds to the compiler commands. It's enabled more often than not in CMakeLists because static libraries built without `-fPIC` cannot be linked into shared libraries. The standard `fPIC` option in conanfiles also sets this value to true automatically as well. It also has some minor security benefits, I suppose, and can have a tiny performance overhead.